events/nickname_events/wc_nickname_events.txt

```diff
@@ -0,0 +1,141 @@
+###################
+# Nickname Events
+###################
+
+namespace = WCNIC
+
+### the Old Murkeye
+WCNIC.0001 = { # Nickname for when the character rule long
```

Comment:
Huh?
The event should be rarer probably? The good 1/3 of rulers above 45 are called the Old. And keep in mind they could just refuse the nickname. So like half of the rulers got this event.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/109849021-55099100-7c6a-11eb-9807-0399b2763ee2.png)
![image](https://user-images.githubusercontent.com/46384992/109849058-5f2b8f80-7c6a-11eb-9bfc-38ccaa168a54.png)

</details>

Reply:
done in 5f28b669